lib/babe/epoch_handler.go

```diff
@@ -63,6 +62,7 @@ func newEpochHandler(epochNumber, firstSlot uint64, epochData *epochData, consta
 }
 
 func (h *epochHandler) run(ctx context.Context, errCh chan<- error) {
```

Comment:
Just gonna put a note on this line, and say that the `errCh` should be a return variable since it's the `epochHandler` that produces to this channel.